quickmenu/arm9/source/graphics/fontHandler.cpp

```diff
@@ -101,6 +102,26 @@ void clearText() {
 	clearText(false);
 }
 
+void printTopSmall(int xPos, int yPos, std::string_view str) {
+	toncset16(FontGraphic::textBuf[1], 0, 256 * smallFont->height());
+	smallFont->print(0, 0, true, str, Alignment::left, FontPalette::regular);
+	int width = smallFont->calcWidth(str);
+
+	xPos -= width*0.5f;
```

Comment:
Why do we use float operatation on integer vars?
How about devide by 2?

Reply:
Just did a clean up of the code. The PR #2398 fixes this.